public/src/main/java/com/revenuecat/purchases/errors.kt

```diff
@@ -50,5 +50,7 @@ enum class PurchasesErrorCode(val code: Int, val description: String) {
     LogOutWithAnonymousUserError(22, "Called logOut but the current user is anonymous."),
     ConfigurationError(23, "There is an issue with your configuration. Check the underlying error for more details."),
     UnsupportedError(24, "There was a problem with the operation. Looks like we doesn't support " +
```

Comment:
This was already here, but:
```suggestion
    UnsupportedError(24, "There was a problem with the operation. Looks like we don't support " +
```